src/javascript/app/App/app.jsx

```diff
@@ -14,12 +13,15 @@ import MarketUnavailableModal      from './Containers/MarketUnavailableModal';
 import ServicesErrorModal          from './Containers/ServicesErrorModal';
 import UnsupportedContractModal    from './Containers/UnsupportedContractModal';
 import Wip                         from './Containers/Wip';
-
+import './i18n';
+import ReactDOM                    from "react-dom";
+import initStore                   from "./app.js";
+import '../../../sass/app.scss';
```

Comment:
Sass Loader! My lord! Thank you!

Reply:
`sass-loader` is implemented, but needs an entry scss file to be imported.